powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehaviorTest.java

```diff
@@ -65,13 +64,20 @@ class ActivationServiceBehaviorTest {
     @Autowired
     private ActivationServiceBehavior tested;
 
-    @Autowired
-    private PowerAuthService powerAuthService;
+    private final ApplicationServiceBehavior applicationServiceBehavior;
+    private final RecoveryServiceBehavior recoveryServiceBehavior;
 
     private final KeyConvertor keyConvertor = new KeyConvertor();
     private final ObjectMapper objectMapper = new ObjectMapper();
     private final String version = "3.2";
     private final String userId = UUID.randomUUID().toString();
+    @Autowired private ActivationServiceBehavior activationServiceBehavior;
```

Comment:
Why do we autowire the field and also the constructor?

Reply:
This is weird...